.github/workflows/tests.yml

```diff
@@ -89,13 +91,16 @@ jobs:
           $SKIP_INIT || init_databases
           $SKIP_INIT || install_glpi
           $SKIP_INIT || init_plugin
+          $SKIP_INIT || install_plugin_dependencies
```

Comment:
in this shell function, plugins are downmoaded and their dependencies installed (composer)